### PR TITLE
Maxim.brown/apm fixes

### PR DESCRIPTION
**Pull request by z0331**
### What does this PR do?
Initial purpose was to fix a couple small typos in the navigation bar under APT (Tracing) --> Setup APM. This expanded slightly as I found a few other small things to fix within the APM documentation.

### Motivation
Just small corrections/fixes. "Librairies" -> "Libraries" "Setup" when used as a verb -> "Set up" and a few others.

### Preview link
https://docs-staging.datadoghq.com/maxim.brown/apm-fixes/

### Additional Notes

- One of the places I fixed "librairies" was in `config.yaml` under `identifier: customnav_tracingnav_libraries`. I don't think this should negatively effect anything else, but I wanted to call it out in case.

- In APM (Tracing) --> Set up APM --> Environment, I removed what I think is an extra 'D' in `-Ddd.trace.span.tags=”env:prod”`. I believe this is correct based on the Java documentation.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -388,7 +388,7 @@ languages:
           weight: 80
         # TRACING/APM - Subnav start
         - identifier: customnav_tracingnav_setup
-          name: "Setup APM"
+          name: "Set up APM"
```
Comment (AndrewMcBurney) [inline config.yaml]: Why did you change it from `Setup` to `Set up`?

File: content/tracing/setup/environment.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ There are several ways to specify an environment when reporting data:
   * **Java**:  
       Via sysprop: 
       ```
-      -Ddd.trace.span.tags=”env:prod”
+      -dd.trace.span.tags=”env:prod”
```
Comment (irabinovitch) [inline content/tracing/setup/environment.md]: intentional?
